package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "npm-blue-tape",
+  "description": "The type definition for https://github.com/spion/blue-tape",
+  "version": "0.0.1",
+  "author": "blakeembrey",
```

Comment:
Yep, looks good. It's up to you on the repo/bugs fields too. And there's the empty `dependencies`. So yeah, not much left :smile: